docs/admin/known_issues.rst

```diff
@@ -2,3 +2,29 @@ Limitations and known issues
 ============================
 
 .. include:: ../includes/top-warning.rst
+Reporting issues
+----------------
+
+For the duration of the pilot, bugs and other issues found in 
+*SecureDrop Workstation* should be reported by pilot participants via the 
+`support portal <https://support.freedom.press>`_. Issues that are not instance-
+specific will be added to the appropriate public issue tracker by the developers, 
+for example:
+
+ - `SecureDrop Workstation issues <https://github.com/freedomofpress/securedrop-workstation/issues>`_ - issues related to the Qubes environment and workstation provisioning.
+ - `SecureDrop Client issues <https://github.com/freedomofpress/securedrop-client/issues>`_ - issues related to the *SecureDrop Client*.
+
+ - `SecureDrop Export issues <https://github.com/freedomofpress/securedrop-export/issues>`_ - issues related to printing and exporting submissions. 
+
+Current known issues
+--------------------
+
```

Comment:
Is it worth adding any information about Qubes known issues (for example: fedora 30 template size), or adding an "if you experience any Qubes-related issues, file a ticket with us" prompt? This isn't a blocker btw just a thought.

Reply:
I'd probably try to keep it scoped to client issues but if it directly impacted us, sure.